scripts/services/check-removed-services.js

```diff
@@ -27,7 +29,7 @@ const getBlockedServicesData = async (filePath) => {
         const serviceObjects = JSON.parse(fileContent);
         return serviceObjects.blocked_services;
     } catch (error) {
-        console.error('Error while reading services.json', error);
+        logger.error(`Error while reading file ${filePath}: ${error.message}`);
         return null;
```

Comment:
it will log the error but also will not restore the services during `restoreRemovedInputServices()`:
```
if (!Array.isArray(blockedServices)) {
   return;
}
```

maybe worth error throwing here to fail script execution more explicitly?